components/dotaReplay.py

```diff
@@ -2,52 +2,164 @@
 import shelve
 from datetime import datetime
 
-async def check_recent_matches(channel, player_list, OPENDOTA_API_KEY):
-    s = shelve.open('./database/george_token_count.db')
-    if s.get("token_count") is None:
-        s["token_count"] = 0
 
+m_open_dota_api_key = ""
+
+def get_arg_list(message, expected_num_args: int, strict: bool):
```

Comment:
I assume this is going to utils.py. Good to have a detailed comment on what the function returns, parameters, etc.

Reply:
Very pog function tho